hs-bindgen/src-internal/HsBindgen/Backend/SHs/Translation.hs

```diff
@@ -287,6 +293,338 @@ translateStorableInstance struct Hs.StorableInstance{..} mbComment = do
       , instanceComment = mbComment
       }
 
+translatePrimInstance ::
```

Comment:
Let's move all of this to `SHs/Translation/Prim` as well. I don't think we've started splitting the `Hs` to `SHs` translation just yet, but we might as well.